src/Liquid/Template.php

```diff
@@ -227,12 +227,13 @@ public function parseFile($templatePath)
 	 * @param array $assigns an array of values for the template
 	 * @param array $filters additional filters for the template
 	 * @param array $registers additional registers for the template
+	 * @param array $tickFunction tick function
 	 *
 	 * @return string
 	 */
-	public function render(array $assigns = array(), $filters = null, array $registers = array())
+	public function render(array $assigns = array(), $filters = null, array $registers = array(), callable $tickFunction = null)
```

Comment:
Same goes here about BC breaks. If someone happen to extend `Template` while adding an argument, they're up for an unpleasant surprise unless we bump a major version. Which is what I would like to avoid.